src/main/java/uk/gov/hmcts/divorce/ccd/NoFaultDivorce.java

```diff
@@ -20,7 +20,7 @@
 @Component
 public class NoFaultDivorce implements CCDConfig<CaseData, State, UserRole> {
 
-    public static final String CASE_TYPE = "NO_FAULT_DIVORCE16";
+    public static final String CASE_TYPE = "NO_FAULT_DIVORCE17";
```

Comment:
Not sure this needs updating. I thought it needed to be a backward incompatible change. Seeing as you are just adding new states should be fine. I may be wrong though 😬

Reply:
This does not need updating if you are adding new state